userguide/system.rst

```diff
@@ -1113,9 +1117,9 @@ The credentials to provide secure connections with cloud services
 are entered here. Amazon S3, Azure Blob Storage, Backblaze B2, and
 Google Cloud Storage are supported.
 
-Navigate
-:menuselection:`System --> Cloud Credentials` and click |ui-add| to open
-the generic Cloud Credential form. Choose a :guilabel:`Provider` to
+Click
+:menuselection:`System --> Cloud Credentials` and |ui-add| to add a
+cloud credential. Choose a :guilabel:`Provider` to
 display any specific options for that provider.
 
```

Comment:
s/to add a cloud credential./to add a new cloud service provider/
s/Choose a :guilabel:`Provider` to display any specific options/Click the :guilabel:`Provider` drop-down to display specific options/